build.sc

```diff
@@ -109,12 +106,25 @@ object `bootstrap-launcher` extends BootstrapLauncher { self =>
       }
     Seq(PathRef(dest))
   }
-  def sources = T.sources {
+  def windowsAnsiPsSources = T {
```

Comment:
It might be possible to use standard dependencies here. We just have to make sure that they end up in the various assemblies we generate around [here](https://github.com/coursier/coursier/blob/13b6faa5936b838d73ba773ba90a23ec5574997f/project/modules/bootstrap-launcher0.sc#L85-L171) (I think they should end up there out-of-the-box… The use of source dependencies is, among others, a remnant of me not wanting to take the time to set things up in sbt, when coursier was built with sbt. This is not an issue anymore, since the switch to Mill.)